### DISCUSSION TITLE
posts/2-years-of-work/

### DISCUSSION THREAD

**Discussion by giscus** [jerrylususu/jerrylususu.github.io#33] [Blog Comments]
# posts/2-years-of-work/

总体感受：7/10

https://nekonull.me/posts/2-years-of-work/

**Comment by zhihui-Yu** [comment]
做开发(四年)的常态，向上不明朗，动力缺乏，总是各个领域涉足，却不够深入。follow you

**Comment by sweet2honey** [comment]
po 主是哪个行业的呀。我也是工作三年了，不愿被同化成拉通怪所以不被待见，搞技术也没有什么项目和机会 :(

**Reply by jerrylususu** [reply]
互联网

**Comment by ZjuerEmma** [comment]
本科制药，搞过咨询，误入餐饮，撩过教育，现在30+转到曾经最讨厌和最不擅长的互联网行业3年了，4个月前看了你的软考文章，受到鼓舞，努力考过了系统分析师，很开心^ ^祝你未来工作顺利，生活满意~
